src/src/rx_main.cpp

```diff
@@ -600,24 +604,16 @@ static void ICACHE_RAM_ATTR updateDiversity()
 
 void ICACHE_RAM_ATTR HWtimerCallbackTock()
 {
+    PFDloop.intEvent(micros()); // our internal osc just fired
+    if (!didFHSS) didFHSS = HandleFHSS();
```

Comment:
Sometimes calling HandleFHSS() before the next if() (which is for DVDA), can induce a little packet jitter.  What is the time difference having it after?

Reply:
Sure I can move it down, but in the case of DVDA (ESP32) the two lines in if block actually just end up setting a boolean which is not going to be serviced in the main loop till the entire interrupt handler is done anyway!

Reply:
We can't move it down as on ESP8285 this is the source of the delay!

Reply:
I found an extra `RXnb` that we were doing at the end of FHSS which was completely unnecessary as we are in continuous RX mode anyway, and after TX (telemetry) we put it back into RX mode. This actually saves a bit of time which means the FHSS call can remain after the `if` statement like you prefer 💪